packages/admin-next/dashboard/src/routes/orders/order-detail/components/order-activity-section/order-timeline.tsx

```diff
@@ -686,3 +712,68 @@ const ExchangeBody = ({
     </div>
   )
 }
+
+const OrderEditBody = ({
+  edit,
+  itemsMap,
+}: {
+  edit: AdminOrderChange
+  isRequested: boolean
+  itemsMap: Record<string, AdminOrderLineItem>
+}) => {
+  const { t } = useTranslation()
+
+  const [itemsAdded, itemsRemoved] = useMemo(
+    () => countItemsChange(edit.actions, itemsMap),
+    [edit]
+  )
+
+  return (
+    <div>
+      {itemsAdded > 0 && (
+        <Text size="small" className="text-ui-fg-subtle">
+          {t("labels.added")}: {itemsAdded}
+        </Text>
+      )}
+
+      {itemsRemoved > 0 && (
+        <Text size="small" className="text-ui-fg-subtle">
+          {t("labels.removed")}: {itemsRemoved}
+        </Text>
+      )}
+    </div>
+  )
+}
+
+function countItemsChange(
+  actions: AdminOrderChange["actions"],
+  itemsMap: Record<string, AdminOrderLineItem>
+) {
+  let added = 0
+  let removed = 0
+
+  actions.forEach((action) => {
+    if (action.action === "ITEM_ADD") {
```

Comment:
I think we could have a const/enum with all the actions we check everywhere, if there are other checks.

Reply:
yup, we have these constants hardcoded in a few places, will refactor this in a separate PR